tox.ini

```
@@ -0,0 +1,25 @@
+[tox]
+envlist = py38,lintcheck
+skipsdist = True
+
+[testenv]
+deps = -rrequirements-dev.txt
+
+[testenv:lintcheck]
+deps =
+    isort
+    black
+    flake8
+commands =
+    isort -c -l 88 --diff .
+    black  --check .
+    flake8 .
+
+[isort]
```

MAY: black のprofileは追加しなくて大丈夫ですか？（前回指摘にコメント等なくoutdatedになってたので念のため）
https://pycqa.github.io/isort/docs/configuration/profiles/
```suggestion
[isort]
profile = "black"
```

---

忘れてました。すいません ><